### PR TITLE
DCS-654 delete prepost appointment

### PR DESCRIPTION
**Pull request by GurnankCheema**

### CODE REVIEW

File: backend/controllers/appointments/prepostAppoinments.js [hunk 1]
```diff
@@ -1,451 +0,0 @@
...
-        court: courtValue,
-      })
-
-      if (errors.length) {
-        packAppointmentDetails(req, appointmentDetails)
-
-        const courtDropDownValues = await getCourtDropdownValues(res.locals)
-
-        if (otherCourtForm) {
-          return res.render('enterCustomCourt.njk', {
```
Comment (andrewrlee) [inline backend/controllers/appointments/prepostAppoinments.js:308]: this refers to other nunjucks templates - can they be removed too or are they used elsewhere?

File: backend/controllers/appointments/prepostAppoinments.js [hunk 2]
```diff
@@ -1,451 +0,0 @@
...
-          location: locationDescription,
-          date: moment(startTime, DATE_TIME_FORMAT_SPEC).format('D MMMM YYYY'),
-          courtHearingStartTime: Time(startTime),
-          courtHearingEndTime: endTime && Time(endTime),
-        },
-      })
-    } catch (error) {
-      logError(req.originalUrl, error, serviceUnavailableMessage)
-      res.render('error.njk', {
-        url: authSource === 'nomis' ? `/offenders/${offenderNo}/add-appointment` : '/videolink/prisoner-search',
```
Comment (andrewrlee) [inline backend/controllers/appointments/prepostAppoinments.js:192]: this is a bit weird, suggests it could be used by more than one place?

Comment (andrewrlee) [reply backend/controllers/appointments/prepostAppoinments.js:192]: weird but I think it might be fine

**Review by andrewrlee — APPROVED** [review]
LGTM

File: backend/controllers/appointments/prepostAppoinments.js (hunk 1 above)
Comment (GurnankCheema) [reply backend/controllers/appointments/prepostAppoinments.js:308]: will delete in next ticket

File: backend/controllers/appointments/prepostAppoinments.js (hunk 2 above)
Comment (louisenorris) [reply backend/controllers/appointments/prepostAppoinments.js:192]: Saw error.njk in use in other files which we think we need so leaving that one for now.

**Pull request merged**